docs-source/content/userguide/cicd/choose-an-approach.md

```diff
@@ -10,44 +10,69 @@ Let's review what we have discussed and talk about when we might want to use
 various approaches.  We can start by asking ourselves questions like these:
 
 
-- *Can you make the desired change with a configuration override?*  
-  The WebLogic Kubernetes Operator allows you to inject a number of [configuration
+- *Can you make the desired change with a configuration override or Model in Image config map?*  
+
+  When your domain source type is Domain in PV or Domain in Image,
+  the WebLogic Kubernetes Operator allows you to inject a number of [configuration
   overrides]({{< relref "/userguide/managing-domains/configoverrides/_index.md" >}})
-  into your pods before starting any servers in the domain.  This allows you to use 
-  the same image for multiple
-  different configurations.  A good example would be changing the settings for a data
-  source, for example. You may wish to have a larger connection pool in your production
+  into your pods before starting any servers in the domain.  
+
+  When your domain
+  source type is Model in Image you can 
```

Comment:
FromModel

Reply:
I think that'd be too big of a change.  Domain in Image, etc, is the standard throughout entire doc set, and everyone on the project is already familiar with this terminology.

Reply:
@jshum2479 Please resolve if you agree, or comment otherwise.

Reply:
@rosemarymarano Any comment on this one?

Reply:
@tbarnes-us I don't understand the comment. Johnny is proposing to use some type of acronym for the domain home source types? If that's it, then I would say it's a bad idea.

Reply:
Pretty sure that's what he's asking for. 
I'm going to resolve this one.